shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/datasource/utils/MySQLDataSourceClassNameGenerator.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.infra.datasource.utils;
+
+/**
+ * MySQL data source class name generator.
+ */
+public final class MySQLDataSourceClassNameGenerator implements DataSourceClassNameGenerator {
+    
+    @Override
+    public String getDataSourceClassName() {
+        return "com.mysql.jdbc.jdbc2.optional.MysqlDataSource";
```

Comment:
Is class `com.mysql.jdbc.jdbc2.optional.MysqlDataSource` enough? There are more than one MySQL driver classes.